libutils/csv_parser.c

```diff
@@ -344,7 +344,11 @@ char *GetCsvLineNext(FILE *fp)
 
         WriterWriteChar(buffer, current);
 
-        if ((current == '\n') && (prev == '\r'))
+        if (current == '\"')
```

Comment:
Any reason for the backslash?

Reply:
Instinctual. Is it more or less readable?
```
'\"'
'"'
```

Reply:
Changed it, @nickanderson seems to side with you.